virtualmachine/virtualmachine.go

```diff
@@ -24,7 +24,7 @@ type VirtualMachine interface {
 	GetSSH(ssh.Options) (ssh.Client, error)
 }
 
-type NetworkSettings struct {
+type NetworkSetting struct {
```

Comment:
Will it require corresponding halo->appos changes in payload?
If so, please mention that in PR template. Open corresponding PRs in link(mention in PR template) to this PR>

Reply:
Yes this will require change in Halo and Appos. Have added link to Halo PR.
Currently working on c3ntry changes and will update that also once it is raised.

Reply:
Have updated PR template to include corresponding Appos PR link.